Application/Items/Commands/ApplyForItem.cs

```diff
@@ -19,8 +19,8 @@ public class ApplyForItem
     {
         public class Command : IRequest
         {
-            public Guid Id { get; set; }
             public string User { get; set; }
+            public Guid Item { get; set; }
```

Comment:
Rename to ItemId, simply calling it Item is confusing and implies the Item object

Reply:
alright